Nos/Views/New Note/ReplyPreview.swift

```diff
@@ -0,0 +1,47 @@
+import SwiftUI
+
+/// This is a component of `NewNoteView` that displays a preview of the note being replied to. 
+struct ReplyPreview: View {
+    
+    var note: Event
+    @Environment(CurrentUser.self) var currentUser
+    
+    var body: some View {
+        VStack(spacing: 8) {
+            HStack {
+                AvatarView(imageUrl: note.author?.profilePhotoURL, size: 30)
+                PlainText(note.author?.safeName ?? "")
+                    .font(.clarity(.medium, textStyle: .subheadline))
+                    .foregroundColor(.primaryTxt)
+                    .bold()
```

Comment:
I'm assuming the `.bold()` overrides the `.medium` but don't know for sure. I'd expect this if we want a bold font weight:

```suggestion
                    .font(.clarity(.bold, textStyle: .subheadline))
                    .foregroundColor(.primaryTxt)
```

Or if we want a medium weight, I'd expect that instead of `.bold` the above `.font` line.